Gemfile

```diff
@@ -39,14 +43,18 @@ gem 'active_record_upsert', '0.11.2'
 gem 'friendly_id'
 
 # HTML Pipeline and dependencies
-gem 'html-pipeline'
+gem 'html-pipeline', '2.14.3'
```

Comment:
Remove the version locks. Imposing a version lock means we will no longer be notified of future upstream updates.  
You can upgrade the gems one-by-one to avoid unnecessary upgrades.

Reply:
Done